profiles/common/modules/custom/tmgmt_poetry/tmgmt_poetry_mock/src/Mock/PoetryMock.php

```diff
@@ -541,4 +614,31 @@ public static function prepareReferenceNumber($demande_id) {
       . '_' . $demande_id['produit'];
   }
 
+  /**
+   * Returns 'Demande ID' array based on given job reference.
```

Comment:
The different items in the array together form the demande_id, as explained also in the documentation of Poetry itself. See also explanation of 'demandeid' on page 33 of the PDF manual linked behind "Poetry services documentation" at https://webgate.ec.europa.eu/CITnet/confluence/display/NEXTEUROPA/WCM+-+Poetry+-+Developer+notes